### DISCUSSION TITLE
languages counter

### DISCUSSION THREAD

**Discussion by fratedesc** [nodeschool/discussions#2948] [Q&A] [answered]
I currently have a repository on GitHub where only one language is showing in the 'Languages' section, even though I've used more than one languages in my code, for example Java & SQL. In my case, only java appears as used language. Is there a way to include these additional languages so the percentages accurately reflect the languages used in the project?

**Comment by ngljcb** [comment, marked as answer]
make sure your files have the right extension which is how github detects the languages, or in your case, you can try to manually specify some languages in `.gitattributes` file, particolarly, you can try this code:

```
# gitattributes
*.sql linguist-detectable
```

**Reply by fratedesc** [reply]
it works! thanks!
